web/src/main/java/com/ritense/valtimo/web/autoconfigure/SwaggerAutoConfiguration.java

```diff
@@ -58,43 +50,35 @@ public class SwaggerAutoConfiguration {
      * @return the Swagger Springfox configuration
      */
     @Bean
-    @ConditionalOnMissingBean(name = "swaggerSpringfoxDocket")
-    public Docket swaggerSpringfoxDocket(SwaggerProperties swaggerProperties) {
+    @ConditionalOnMissingBean(name = "valtimoOpenAPI")
+    public OpenAPI valtimoOpenAPI(SwaggerProperties swaggerProperties) {
```

Comment:
Shouldn't the SwaggerProperties class be renamed as well?

Reply:
I suppose while i'm working on this i could rename all swagger related things to OpenAPI.